src/Components/Web.JS/src/Boot.WebView.ts

```diff
@@ -13,6 +14,16 @@ async function boot(): Promise<void> {
   }
   started = true;
 
+  const jsInitializersResponse = await fetch('_framework/blazor.modules.json', {
+    method: 'GET',
+    credentials: 'include',
+    cache: 'no-cache'
+  });
+
+  const initializers: string[] = await jsInitializersResponse.json();
+  const jsInitializer = new JSInitializer();
+  await jsInitializer.importInitializersAsync(initializers, []);
```

Comment:
Any chance of moving the repeated server/webview code (and possible more of the webassembly code) out into a separate place?

Apologies if this is a bit fussy, but these initialization files just keep on growing with every new feature and I feel like I have to try to hold back the tide on it. The WebView one was pretty small and clean and would be great to keep like that.

Reply:
Hopefully would only take a few mins, as you already have the `JSInitializers.ts` file and can export another function from it.

Reply:
I thought I did this, do you have something even more 'minimalistic' in mind?

It's 4 lines total compared to 2 lines (which is the min I can reduce it to)